google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -535,7 +538,8 @@ TEST_F(DataTypeIntegrationTest, SelectIntervalFromTimestampDiff) {
           std::chrono::hours(1))};
   std::time_t now_seconds =
       std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
-  std::time_t one_hour_later_seconds = now_seconds + 3600;
+  // Use chrono_literals for 1 hour
+  std::time_t one_hour_later_seconds = now_seconds + std::chrono::duration_cast<std::chrono::seconds>(1h).count();
```

Comment:
`.count()` is only a convoluted way to write an integer.  The issue here is that chrono arithmetic should be done in the chrono domain.  No raw numbers.

For example, consider something as simple as:
```
  auto const t0 = std::chrono::system_clock::now();
  auto const t1 = t0 + std::chrono::hours(1);
  std::vector<std::vector<Timestamp>> const data = {
      {MakeTimestamp(t0).value(), MakeTimestamp(t1).value()}};
  auto const expected_interval = Interval(t1 - t0);
```